rust/src/ftp/response.rs

```diff
@@ -22,11 +22,12 @@ use std::slice;
 
 #[repr(C)]
 pub struct FTPResponseLine {
-    code: *mut u8,     // Response code as a string (may be null)
```

Comment:
Makes me want to have `code: Vec<u8>` with some FFI function to get the pointer and length to C...

Reply:
I don't see an example of this anywhere.